Dec 4, 2024 TAC Minutes (#63)
PQCA/TAC: meeting-minutes/2024/2024-12-04-TAC.md

* Joe Livingston
---
## Introduction of 1st time attendees
* None

---
## Announcements
* [Nvidia cuPQC release](https://developer.nvidia.com/cupqc)

---
## OQS Update -Spencer Wilson
* Initial feedback on 0.12.0 from users are good.
* open [discussion on API](https://github.com/open-quantum-safe/liboqs/issues/2001)
* TSC meeting will cover security and slh-dsa 
* looking for additional slh-dla expertise
 
---
## PQCP Update -Nigel Jones
* [mlkem-native v1.0.0-alpha](https://github.com/pq-code-package/mlkem-native/releases/tag/v1.0.0-alpha) 
* blog for release is in the works
* [cross-functional API discussion](https://github.com/pq-code-package/tsc/issues/4) 
* no external security audit of code currently scheduled, but having discussions on when to do that. For now the focus is on on being transparent about where things stand. 

---
## Tools WG Update -Nicklas Kortge
* creating PRs for bigger, open source repos, where we may want to have BOMs available
*  participation is still on the light side as things ramp up.
---

## API differences between the draft standards and the final versions, and the plight of prehashing ML-DSA - Sophie Schmieg
* Sophie shared a presentation 

> * NIST standardized 4 schemes resulting in breaking changes to the API
> * Suggests added a new context string to the API

Reviewer: ```suggestion
* NIST standardized 4 schemes
* Final standards added a new context string to the signature API, which is a breaking change
```
Suggest clarifying this as the KEM API did not change.

Reviewer: It might also be worth adding a bullet point here about Sophie's thoughts on HashML-DSA. Something like "* Purpose of adding HashML-DSA in the final standard unclear"

* ACTION provide this same talk to PQCP TSC coordinated w/ Nigel

---
## Open Action Items / Issues Review
* recently closed documentation discussions - need to ensure should be opened as a new issue
* Max will respond to [issue 42](https://github.com/PQCA/TAC/issues/42) and close it
* [issue 59](https://github.com/PQCA/TAC/issues/59) Brian will close
* [issue 61](https://github.com/PQCA/TAC/issues/61)  - need to allow reviewers a week to review and when clock runs out it gets published 

> * [issue 45](https://github.com/PQCA/TAC/issues/45) Do in OQS first and then make recommendations to roll out further.

Reviewer: ```suggestion
* [issue 45](https://github.com/PQCA/TAC/issues/45) Move issue to OQS for possible adoption before potentially being rolled out further
```
I suggest adding these qualifiers to make it clear that the action item was not to start working on the issue in OQS but rather to rehome the issue to an OQS repo.
